ecosystem/sep-0010.md

```diff
@@ -18,12 +18,21 @@ This SEP defines the standard way for clients such as wallets or exchanges to cr
 
 This protocol is a variation of mutual challenge-response, which uses Stellar transactions to encode challenges and responses.
 
-The authentication flow is as follows:
+It involves three parties:
+- A home domain hosting a [SEP-1 stellar.toml](sep-0001.md) containing a `WEB_AUTH_ENDPOINT` value.
+- A web auth endpoint providing the GET and POST operations discussed in this document.
+- A client who will authenticate with a Stellar account.
+
+The discovery flow is as follows:
+
+1. The client retrieves the [SEP-1 stellar.toml](sep-0001.md) from the home domain. The home domain hosts the stellar.toml.
+1. The client looks up the `WEB_AUTH_ENDPOINT` value from the [SEP-1 stellar.toml](sep-0001.md).
 
-1. The client obtains a unique [`challenge`](#challenge), which is represented as specially formed Stellar transaction
+The authentication flow is as follows:
+1. The client obtains a unique [`challenge`](#challenge) from the `WEB_AUTH_ENDPOINT`, which is represented as specially formed Stellar transaction
 1. The client verifies that the transaction has an invalid sequence number 0.  This is extremely important to ensure the transaction isn't malicious.
 1. The client verifies that the transaction is signed by the `SIGNING_KEY` specified by the requested service's [SEP-1 stellar.toml](sep-0001.md).
```

Comment:
```suggestion
1. The client verifies that the transaction is signed by the `SIGNING_KEY` obtained through discovery flow.
  * If this verification fails, client should re-execute the discovery flow and attempts to resume the authentication flow with the same challenge and a freshly retrieved `SIGNING_KEY`. This is necessary to support seamless key rotation for web services. If verification fails the second time, client must not continue the authentication and should present to the user (or write into the log, in case of machine-to-machine authentication) a warning about potential spoofing attack.
```

Reply:
I applied this change but missed the change included adding the additional discussion about key rotation. This PR is focused on adding home domain back and so I've removed the comments about how to do key rotation in bb7700e0bfbbe930f328eafe27f2390875efadf6. I think how to do key rotation is something we should discuss independently.